### ISSUE TITLE
Add more client stats

### ISSUE THREAD

**Issue by stevej** [linkerd/linkerd-tcp#49]
Debugging the mystery of the failing iperf tests would have been easier had `/metrics` shown me that there were other clients connected and where they were connected from.

**Comment by olix0r** [comment]
We probably shouldn't expose any per-endpoint metrics by default unless we have a good way of garbage collecting unused enpdpoints (i.e. we'd need to ensure that dropping an endpoint removes its counters, but only after they've been reported...  it's a bit tricky).

What non-per-endpoint metrics would have helped you?

**Comment by stevej** [comment]
non-per-endpoint metric: Unique number of currently connected clients.

I would have expected 1 when running my load test with a concurrency of 1. Seeing > 1 would have led me to login to the host and use `ss` to see who else was connected.

**Comment by tamird** [comment]
@olix0r this should have been closed by ea507f7.

FYI this doesn't do what you think:
> Fixes #1 #2 #3"

Each issue needs its own "fixes":
> Fixes #1
> Fixes #2
> Fixes #3
